### PR TITLE
Ephemeris debug

### PR DESCRIPTION
**Pull request by denniszollo**
Use the SBP definition of the ephemeris message

### CODE REVIEW

**Comment by denniszollo** [comment]
retest this please

File: src/ephemeris.c [hunk 1]
```diff
@@ -88,8 +88,10 @@ static msg_t nav_msg_thread(void *arg)
       if (!es[ch->prn].healthy) {
         log_info("PRN %02d unhealthy\n", ch->prn+1);
       } else {
-        sbp_send_msg(SBP_MSG_EPHEMERIS, sizeof(ephemeris_t),
-                     (u8 *)&es[ch->prn]);
+        msg_ephemeris_t e;
+        pack_ephemeris(&e, (ephemeris_t *) &es[ch->prn]);
```
Comment (fnoble) [inline src/ephemeris.c]: Why is the cast needed here?

Comment (denniszollo) [inline src/ephemeris.c]: I don't think it is required, I kind of like it for clarity so you don't have to go check the array of structures.

Comment (denniszollo) [inline src/ephemeris.c]: I'm removing and force pushing

**Comment by fnoble** [comment]
lgtm - update libsbp version and merge at will

**Comment by denniszollo** [comment]
I actually cannot merge this one because the requisite changes to SBP went in in the middle of the bootload refactoring which is still ongoing.  I will plan to hold this until after a new SBP version is merged into piksi_firmware.

**Comment by mfine** [comment]
Going to pull this in to #486.

**Comment by mfine** [comment]
Note, this was a packed struct vs non-packed struct (think the ephemeris struct in libswiftnav was largely aligned). This could cause problems interoperating.

**Comment by mfine** [comment]
Brought in with https://github.com/mfine/piksi_firmware/commit/d3d7f5e1ab09f2c25d13a9431425222c2ba275ce.

**Pull request closed without merging**